### PR TITLE
Redo v1.15.1 release again

### PR DESCRIPTION
**Pull request by michi-covalent**
https://github.com/isovalent/olm-for-cilium/pull/84 didn't work. please review https://github.com/isovalent/olm-for-cilium/pull/85/commits/8f2c895745dffd77d663968da08624b330be7d9d 🚀🙏 

✅ manually tested with v1.15.1 and v1.14.7.

### CODE REVIEW

File: config/operator/olm.cue [hunk 1]
```diff
@@ -174,21 +174,14 @@ _related_images: [
...
-	_related_images: _related_images + [{
-		name: "hubble-ui-proxy"
-		image: parameters.hubbleUIProxyImage
-	}]
-}
-
-// quay.io/coreos/etcd is not included in Cilium >= 1.15 Helm charts.
-// Ref: https://github.com/cilium/cilium/blob/v1.15.1/install/kubernetes/cilium/values.yaml
-if strings.HasPrefix(parameters.ciliumVersion, "1.12") || strings.HasPrefix(parameters.ciliumVersion, "1.13") || strings.HasPrefix(parameters.ciliumVersion, "1.14") {
-	_related_images: _related_images + [{
```
Comment (michi-covalent) [inline config/operator/olm.cue:187]: this didn't work, i guess cue doesn't allow mutating lists, i got this error: `cue: marshal error: template.items.14.spec.relatedImages: cycle detected`

File: config/operator/operator.cue [hunk 2]
```diff
@@ -57,18 +57,16 @@ _related_images_env_vars: [{
...
 		name: "RELATED_IMAGE_NODEINIT"
 		value: parameters.nodeInitImage
-	},
-	{
-		name: "RELATED_IMAGE_CLUSTERMESH_ETCD"
-		value: parameters.clustermeshEtcdImage
 	}]
 
-if parameters.hubbleUIProxyImage != "nothing" {
-	_related_images_env_vars: _related_images_env_vars + [{
```
Comment (michi-covalent) [inline config/operator/operator.cue:67]: ditto

File: ciliumconfig.conformance.v1.15.yaml [hunk 3]
```diff
@@ -24,7 +24,7 @@ spec:
       clusterPoolIPv4PodCIDRList:
         - "10.128.0.0/14"
       clusterPoolIPv4MaskSize: "23"
-  nativeRoutingCIDR: "10.128.0.0/14"
+  ipv4NativeRoutingCIDR: "10.128.0.0/14"
```
Comment (michi-covalent) [inline ciliumconfig.conformance.v1.15.yaml:27]: `ipv4NativeRoutingCIDR` is the correct setting: https://docs.cilium.io/en/stable/helm-reference/ not sure where i initially got `nativeRoutingCIDR` from.

Comment (fgiloux) [reply ciliumconfig.conformance.v1.15.yaml:27]: nativeRoutingCIDR has been deprecated and removed see https://github.com/isovalent/olm-for-cilium/pull/74

**Review by fgiloux — APPROVED** [review]

File: ciliumconfig.conformance.v1.15.yaml (hunk 3 above)
Comment (michi-covalent) [reply ciliumconfig.conformance.v1.15.yaml:27]: oh yeah i guess i made the prior PR before https://github.com/isovalent/olm-for-cilium/pull/74 💡

**Pull request merged**